test/fixers/test_jsonnetfmt_fixer_callback.vader

```diff
@@ -0,0 +1,41 @@
+Before:
+  Save g:ale_jsonnet_jsonnetfmt_executable
+  Save g:ale_jsonnet_jsonnetfmt_options
+
+  " Use an invalid global executable, so we don't match it.
+  let g:ale_jsonnet_jsonnetfmt_executable = 'xxxinvalid'
+  let g:ale_jsonnet_jsonnetfmt_options = ''
+
+  call ale#test#SetDirectory('/testplugin/test/fixers')
+
+After:
+  Restore
+
+  call ale#test#RestoreDirectory()
+
+Execute(The jsonnetfmt callback should return the correct default values):
+  call ale#test#SetFilename('../jsonnet_files/testfile.jsonnet')
+
+  AssertEqual
```

Comment:
Instead of AssertEqual we can use the dedicated AssertFixer. Check the help `:h ale-development-fixer-tests` for details on how to test fixers.